src/mondeath.cpp

```diff
@@ -97,6 +97,37 @@ void mdeath::boomer(monster *z)
     }
 }
 
+void mdeath::boomer_glow(monster *z)
+{
+    Creature *ch = ( z->get_killer() );
```

Comment:
Note: killer can be null (when a bomb explodes, a car rams the zed to death, zed dies in fire).